h/api/events.py

```diff
@@ -4,9 +4,10 @@
 class AnnotationEvent(object):
     """An event representing an action on an annotation."""
 
-    def __init__(self, request, annotation, action):
+    def __init__(self, request, annotation_id, serialized_annotation, action):
         self.request = request
-        self.annotation = annotation
+        self.annotation_id = annotation_id
+        self.serialized_annotation = serialized_annotation
```

Comment:
Is `annotation_id` needed? Or can code just use `serialized_annotation["id"]`? I guess it's not doing any harm though and it's convenient